android/src/com/mapswithme/util/statistics/Statistics.java

```diff
@@ -237,6 +246,22 @@ public void trackSimpleNamedEvent(String eventName)
     post(eventName);
   }
 
+  public void trackConnection()
+  {
+    if (ConnectionState.isConnected())
+    {
+      final NetworkInfo info = ConnectionState.getActiveNetwork();
+      final boolean isConnectionMetered = ((ConnectivityManager) MwmApplication.get().getSystemService(Context.CONNECTIVITY_SERVICE)).isActiveNetworkMetered();
+      //noinspection ConstantConditions
+      post(EventName.ACTIVE_CONNECTION,
+           new String[]{EventParam.CONNECTION_NAME, info.getTypeName() + ":" + info.getSubtypeName(),
+               EventParam.CONNECTION_FAST, String.valueOf(ConnectionState.isConnectionFast(info)),
+               EventParam.CONNECTION_METERED, String.valueOf(isConnectionMetered)});
+    }
+    else
+      post(EventName.ACTIVE_CONNECTION, new String[]{EventParam.CONNECTION_NAME, ""});
```

Comment:
Is empty parameter name OK for Flurry? May be "null" or "Not Connected" would be easier to read?